Description: Remove ext directory
 rainbow 2.2.2 introduced an 'ext' folder which makes gem2deb 
 think it is a native extension and creates an empty directory
 causing lintian to report an error. Thanks to terceiro
Author: Abhijith PA <abhijith@openmailbox.org>
Forwarded: no
Last-Update: 2017-09-12

--- ruby-rainbow-2.2.2.orig/rainbow.gemspec
+++ ruby-rainbow-2.2.2/rainbow.gemspec
@@ -22,5 +22,5 @@ Gem::Specification.new do |spec|
   spec.add_dependency "rake"
 
   spec.add_development_dependency "bundler", "~> 1.3"
-  spec.extensions    = ["ext/mkrf_conf.rb"]
+  spec.extensions    = []
 end
